packages/indexer/src/index.ts

```diff
@@ -2,55 +2,32 @@ export * from './indexer.gen'
 
 import { Indexer as IndexerRpc } from './indexer.gen'
 
-// TODO: rename to SequenceIndexerNetworks
-export enum SequenceIndexerServices {
-  MAINNET = 'https://mainnet-indexer.sequence.app',
-
-  POLYGON = 'https://polygon-indexer.sequence.app',
-  POLYGON_MUMBAI = 'https://mumbai-indexer.sequence.app',
-
-  POLYGON_ZKEVM = 'https://polygon-zkevm-indexer.sequence.app',
-
-  ARBITRUM = 'https://arbitrum-indexer.sequence.app',
-  ARBITRUM_NOVA = 'https://arbitrum-nova-indexer.sequence.app',
-
-  OPTIMISM = 'https://optimism-indexer.sequence.app',
-  AVALANCHE = 'https://avalanche-indexer.sequence.app',
-  GNOSIS = 'https://gnosis-indexer.sequence.app',
-
-  BSC = 'https://bsc-indexer.sequence.app',
-  BSC_TESTNET = 'https://bsc-testnet-indexer.sequence.app',
-
-  GOERLI = 'https://goerli-indexer.sequence.app'
-}
-
```

Comment:
I removed these and SequenceIndexerServices below as they seem unnecessary and aren't used in webapp. I asked Deno if it's used on builder or marketplace. Will put them back in case they are used there.

Reply:
update: Deno confirmed they are not used in builder or marketplace, so it should be ok